modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
+:context: ossm-deploy-application-workloads-per-user-group
+
+This procedure demonstrates how to deploy application workloads in separate namespaces based on user groups.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Label the namespace with the respective control plane:
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 usergroup=usergroup-1 istio.io/rev=usergroup-1
+----
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-1
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml -n app-ns-1
+----
+
+. Wait for the `httpbin` and `sleep` pods to run with sidecars injected:
++
+[source,terminal]
+----
+$ oc get pods -n app-ns-1
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-7f56dc944b-kpw2x   2/2     Running   0          2m26s
+sleep-5577c64d7c-b5wd2     2/2     Running   0          91m
+----
+
+. Repeat the above steps for two more namespaces: `app-ns-2` and `app-ns-3`.
```

Comment:
```suggestion
. Repeat the above steps for two more namespaces: `app-ns-2` and `app-ns-3`, adding them both to the `usergroup-2` revision.
```
It wasn't clear where these should be created otherwise.